model_api/python/model_api/adapters/onnx_adapter.py

```diff
@@ -122,6 +124,9 @@ def await_all(self):
     def await_any(self):
         pass
 
+    def get_raw_result(self, infer_result):
```

Comment:
We need to annotate these as well but it something for later when we refactor the result types.